tools/bumper/cnaoRepoCommands.go

```diff
@@ -67,6 +68,30 @@ func (cnaoRepoOps *gitCnaoRepo) updateComponentsConfig(relativeConfigPath string
 	return updateComponentsYaml(configPath, componentsConfig)
 }
 
+// resetRepoInAllowedList resets all the files in AllowedList.
+func (cnaoRepoOps *gitCnaoRepo) resetRepoInAllowedList(allowList []string) error {
```

Comment:
I now this functions is good for unit test since you pass the allowList, but can we create also the following to encapsulate things a little more ? 

```golang
func (cnaoRepoOps *gitCnaoRepo) resetRepo()  error {
    return resetRepoInAllowedList(allowList)
}
```

Reply:
Like! =)